CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### New Features
 
+- add dynamic sequence scope for matching nearby calls within a thread #2532 @williballenthin
```

Comment:
naming alternatives to sequence (matching occurs in any order): span, ngram, group/cluster

Reply:
+1 `cluster`

Reply:
"window"

Reply:
math: multiset (or bag, or mset) - https://en.wikipedia.org/wiki/Multiset
* multiple instances of same object
* order doesn't matter

Reply:
optionally prefix with "call", e.g., `callbag`, `callcluster`?